types/coin.go

```diff
@@ -137,21 +137,23 @@ type Coins []Coin
 
 // NewCoins constructs a new coin set.
```

Comment:
```suggestion
// NewCoins constructs a new coin set. The provided coins will be sanitized by removing 
// zero coins and sorting the coin set. A panic will occur if the coin set is not valid. 
```
or something similar. Would be great to update for `ParseCoins` and the DecCoins funcs as well